### PR TITLE
Small tweaks to highlight our ad services

### PR DESCRIPTION
**Pull request by davidfischer**

### CODE REVIEW

**Review by ericholscher — APPROVED** [review]
I think we could be more direct than "let us know" (eg. "we can help with that" or "We're happy to create 2 example ads for your campaign to kick things off"), but this is an improvement already 👍

File: content/pages/advertising-faq.md [hunk 1]
```diff
@@ -163,10 +163,16 @@ After you've collected enough data, turn the lower performing variant off to max
 Absolutely. You can contact us anytime, but an ad specialist will proactively reach out
 a few weeks into your campaign with suggestions.
 We also offer advice on crafting ads that resonate with our audience.
-See our [advertising guide]({filename}/pages/learning-hub/advertiser-guide.md) for more details.
+See our [advertising guide]({filename}/pages/learning-hub/advertiser-guide.md) for more details
+and our [creative specs guide]({filename}/pages/learning-hub/ad-design-and-specs.md) has a number of tips and tricks about what works.
+
+We can work with you to ensure your campaign is successful.
+If you have creatives you're running on other networks, we can help you adapt them to our ad formats.
+If you need more help creating ads that convert, let us know.
```
Comment (davidfischer) [inline content/pages/advertising-faq.md:171]:
```suggestion
If you need more help creating ads that convert, we can help with that.
```

File: ethicalads-theme/templates/ea/advertisers.html [hunk 2]
```diff
@@ -223,8 +223,8 @@ <h4 class="font-weight-bold" id="faq-policy">How can I measure ad performance wi
         <p>Use your existing conversion tracking mechanism such as analytics and rely on UTM or ref parameters added to your ad landing page link.</p>
       </div>
       <div class="col-md-6">
-        <h4 class="font-weight-bold" id="faq-customer-service">Will you help me optimize my campaign?</h4>
-        <p>An ad specialist will proactively reach out a few weeks into your campaign with suggestions. We also offer advice on crafting ads that resonate with our audience. For larger campaigns, we create a short video summary of how your campaign is performing with detailed suggestions.</p>
+        <h4 class="font-weight-bold" id="faq-customer-service">Need help with your creatives?</h4>
+        <p>If you have creatives you're running on other networks, we can help you adapt them to our <a href="/advertisers/ad-design-and-specs/">ad formats</a>. If you need more help creating ads that convert, let us know. We've run a lot of campaigns on our network and we know what works.</p>
```
Comment (davidfischer) [inline ethicalads-theme/templates/ea/advertisers.html:227]:
```suggestion
        <p>If you have creatives you're running on other networks, we can help you adapt them to our <a href="/advertisers/ad-design-and-specs/">ad formats</a>. If you need more help creating ads that convert, we can help with that. We've run a lot of campaigns on our network and we know what works.</p>
```

**Review by davidfischer — COMMENTED** [review]
How about this?

**Comment by ericholscher** [comment]
🎉

**Pull request merged**